lib/db/throttler.js

```diff
@@ -16,6 +16,7 @@ class TimeoutThrottler {
       await this._flush()
       this.resolvers.forEach(r => r())
     } catch (err) {
+      console.log(err)
```

Comment:
@akvlad Needed still? Straggler from testing?

Reply:
lets clean it up or move to the new logger error log